# HG changeset patch
# User Rik <rik@octave.org>
# Date 1515340031 28800
#      Sun Jan 07 07:47:11 2018 -0800
# Branch stable
# Node ID e891b6dd7e3527344c78a43bb38a4bd4c5088abe
# Parent  59d080a90f1760a60b5ba8ed6548137bc307359c
doc: Use '...' rather than deprecated '\' for line continuation (bug #52828).

* diffeq.txi: Replace '\' for line continuation outside of double-quoted string
with '...' in example code.

diff --git a/doc/interpreter/diffeq.txi b/doc/interpreter/diffeq.txi
--- a/doc/interpreter/diffeq.txi
+++ b/doc/interpreter/diffeq.txi
@@ -69,17 +69,17 @@ Here is an example of solving a set of t
 
 @example
 @group
 ## oregonator differential equation
 function xdot = f (x, t)
 
   xdot = zeros (3,1);
 
-  xdot(1) = 77.27 * (x(2) - x(1)*x(2) + x(1) \
+  xdot(1) = 77.27 * (x(2) - x(1)*x(2) + x(1) ...
             - 8.375e-06*x(1)^2);
   xdot(2) = (x(3) - x(1)*x(2) - x(2)) / 77.27;
   xdot(3) = 0.161*(x(1) - x(3));
 
 endfunction
 @end group
 @end example
 
@@ -96,17 +96,17 @@ y = lsode ("f", x0, t);
 @end example
 
 If you try this, you will see that the value of the result changes
 dramatically between @var{t} = 0 and 5, and again around @var{t} = 305.
 A more efficient set of output points might be
 
 @example
 @group
-t = [0, logspace(-1, log10(303), 150), \
+t = [0, logspace(-1, log10(303), 150), ...
         logspace(log10(304), log10(500), 150)];
 @end group
 @end example
 
 See @nospell{Alan C. Hindmarsh},
 @cite{ODEPACK, A Systematized Collection of ODE Solvers},
 in Scientific Computing, @nospell{R. S. Stepleman}, editor, (1983)
 for more information about the inner workings of @code{lsode}.
